docs/coolit-guide.md

```diff
@@ -0,0 +1,51 @@
+# Corsair Hydro H110i GT AIO liquid cooler
+_Driver API and source code available in [`liquidctl.driver.coolit`](../liquidctl/driver/coolit.py)._
+
+## Initialization
+[Initialization]: #initialization
+
+The AIO does not need to be initialized prior to use, but it will set the pump
+mode to quiet.
+
+```
+# liquidctl initialize
+Corsair H110i GT (experimental)
+└── Firmware version    2.0.0
+```
+
+When (re)initializing the device, it is possible to select the pump mode:
+
+```
+# liquidctl initialize --pump-mode extreme
+Corsair Hydro H100i Pro
+└── Firmware version      2.10.0.0
```

Comment:
Shouldn't it be just `2.10.0`? The code doesn't output four numbers.

Reply:
Actually `2.0.0` but yeah I changed it